.github/workflows/auto_update_charm_libs.yaml

```diff
@@ -24,14 +24,27 @@ jobs:
           charmcraft fetch-lib
         env:
           CHARMCRAFT_AUTH: ${{ secrets.CHARMHUB_TOKEN }}
+      - name: Import GPG key
+        id: import-gpg
+        uses: crazy-max/ghaction-import-gpg@v5
+        with:
+          gpg_private_key: ${{ secrets.GPG_PRIVATE_KEY }}
```

Comment:
Is this provided by GitHub actions or do we need to set this on each repository?

Reply:
I'm afraid we'll need to set it up for each repository